docs/pages/versions/unversioned/sdk/video.mdx

```diff
@@ -139,6 +139,28 @@ const styles = StyleSheet.create({
 
 </SnackInline>
 
+## Playing local media from the assets directory
+
+`expo-video` does not support playing media directly from a result of calling the `require` function. Use `expo-asset` to load the video before using it as the video source:
+
+```tsx
+import { Asset } from 'expo-asset';
+import { VideoSource } from 'expo-video';
+
+const asset = Asset.fromModule(require('./assets/bigbuckbunny.mp4'));
+
+const bigBuckBunnySource: VideoSource = {
+  // asset.localUri or asset.uri will be used depending on if the app was built in release or debug mode
+  uri: asset.localUri || asset.uri,
```

Comment:
That is very confusing. Can't we call `Asset.loadAsync` instead of `Asset.fromModule`? That should fetch the asset.